wasm/samples/fees_executor/src/lib.rs

```diff
@@ -0,0 +1,125 @@
+//! Iroha executor with fees support.
+
+#![no_std]
+
+#[cfg(not(test))]
+extern crate panic_halt;
+
+extern crate alloc;
+
+use dlmalloc::GlobalDlmalloc;
+use fees_executor_data_model::parameters::*;
+use iroha_executor::prelude::*;
+use iroha_executor_data_model::parameter::Parameter as _;
+
+#[global_allocator]
+static ALLOC: GlobalDlmalloc = GlobalDlmalloc;
+
+#[derive(Visit, Execute, Entrypoints, Debug, Clone)]
+#[visit(custom(
+    visit_set_parameter,
+    visit_unregister_domain,
+    visit_unregister_asset_definition,
+    visit_unregister_account,
+))]
+struct Executor {
+    host: Iroha,
+    context: Context,
+    verdict: Result,
+}
+
+/// Finds currently used fees options
+fn finds_fee_options(host: &Iroha) -> FeesOptions {
+    let parameters = host.query_single(FindParameters).dbg_unwrap();
+
+    let fees_options: FeesOptions = parameters
+        .custom()
+        .get(&FeesOptions::id())
+        .unwrap()
+        .try_into()
+        .expect("INTERNAL BUG: Failed to deserialize json as `FeesOptions`");
```

Comment:
```suggestion
        .expect("fees executor sets FeesOptions during migration and does not allow its modification");
```

Reply:
My thinking is that the message in `.expect()` states the condition, or invariant, of why we _expect_ this to be not a recoverable error. This is self-documenting.

And actually I did a mistake. This `.expect()` is indeed related to the deserialisation error. My message is about the upper `.unwrap()`.

```rs
        .get(&FeesOptions::id())
        .expect("INTERNAL BUG: fees executor sets FeesOptions during migration and does not allow its modification")
        .try_into()
        .expect("`INTERNAL BUG: Failed to deserialize json as `FeesOptions``");
```

Anyway, it really doesn't matter very much what the message is. I'll be fine if you leave it as is.